docs-chef-io/content/inspec/cli.md

```diff
@@ -272,6 +272,8 @@ This subcommand has additional options:
     Read configuration from JSON file (`-` reads from stdin).
 * ``--controls=one two three``
     A list of control names to run, or a list of /regexes/ to match against control names. Ignore all other tests.
+* ``--tags=one two three``
+    A list of tags names that are part of controls to filter and run controls, or a list of /regexes/ to match against tags names of controls. Ignore all other tests. And when tag on the control is a hashmap, it only uses values of hashmap for filtering controls.
```

Comment:
```suggestion
    A list of tags or a list of regular expressions that match tags. `exec` will run controls referenced by the listed or matching tags.
```

Reply:
@IanMadd I feel we should also include in doc that if tags are of type hash then it matches the value of an hash not a key to bring more clarity. I mentioned this sentence for that purpose "And when tag on the control is a hash map, it only uses values of hashmap for filtering controls." Can we put this in more clear way in doc ? What are your suggestions?

Reply:
Maybe something like this: 

A list of tags, a list of regular expressions that match tags, or a hash map where each value is a tag. 
`exec` will run controls referenced by the listed or matching tags.